brainstormforce/astra
# AST-1372 - Added transparent header color options for mode switcher & preview, dynamic support provided

### Description
- Added transparent header color options for mode switcher & preview, dynamic support provided

### Screenshots
- Transparent header color options - https://share.bsf.io/6quvo098

### Types of changes
- New feature (non-breaking change

### How has this been tested?
- Check these color options
- Check with different switch styles
- Check with mode switching layout
- Check preview & frontend correctly loads style

### Checklist:
